release-notes/README.md

```diff
@@ -4,6 +4,21 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## January 24, 2022
+
+### All Unity packages
+
+The ACP Unity packages now use XCFrameworks in order to support hardware with the new Apple M1 architecture while maintaining support for existing Intel architecture. See the respective GitHub repositories for the updated installation instructions.
```

Comment:
This doesn't have to do with this specifically (and doesn't need to be addressed in this PR), but shouldn't the name be AEP Unity? Since, as far as I can tell, the ACP-branding is out of date completely at this point...

Reply:
"... with the new Apple M1 architecture, while maintaining support..."

Reply:
@calebk1 I think using ACP in this context is more appropriate as the Unity packages link to the ACP SDKs version, such as ACPCore etc.